### PR TITLE
Medkit upgrade and sprite port

### PR DESCRIPTION
**Pull request by Alterist**
:cl:
add: Medkits now show how much is left in it's stack
imageadd: updated some item/storage sprites thanks to Bay and CEV-Eris
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -147,6 +147,27 @@
 	stack_id = "Tear petals"
 
 
+/obj/item/stack/medical/advanced/
```
Comment (DominikPanic) [inline code/game/objects/items/stacks/medical.dm]:
```suggestion
/obj/item/stack/medical/advanced
```

File: code/game/objects/items/stacks/medical.dm [hunk 2]
```diff
@@ -147,6 +147,27 @@
...
+	dir = NORTH
+	flags_atom = DIRLOCK
+
+/obj/item/stack/medical/advanced/update_icon()
+	if(max_amount < 1)
+		return
+	var/percentage = round(amount / max_amount) * 100
+	if(amount >= max_amount)
```
Comment (DominikPanic) [inline code/game/objects/items/stacks/medical.dm]: Early return, combine it with the first check.

Comment (Rohesie) [reply code/game/objects/items/stacks/medical.dm]: Is the check even necessary? As it it, it should never be true, so this should only work for full stacks.
If you wanted it not to update the sprite if the amount is higher than the maximum possible, the why set the switch to account for `to INFINITY`?

File: code/game/objects/items/stacks/medical.dm [hunk 3]
```diff
@@ -147,6 +147,27 @@
...
 
+/obj/item/stack/medical/advanced
+	dir = NORTH
+	flags_atom = DIRLOCK
+
+/obj/item/stack/medical/advanced/update_icon()
+	if(max_amount < 1)
+		return
+	var/percentage = round(amount / max_amount) * 100
+	if(max_amount >= amount)
```
Comment (DominikPanic) [inline code/game/objects/items/stacks/medical.dm]: Combine it with the first check for that double sweet early return.

Comment (Rohesie) [reply code/game/objects/items/stacks/medical.dm]: That would be something like `if(max_amount <1 || max_amount < amount)` for an early return.

Comment (Alterist) [reply code/game/objects/items/stacks/medical.dm]: changed

File: code/game/objects/items/stacks/medical.dm [hunk 4]
```diff
@@ -254,7 +276,7 @@
 	stack_id = "splint"
 
 /obj/item/stack/medical/splint/attack(mob/living/carbon/M, mob/user)
-	if(..()) return 1
+	if(..()) return TRUE
```
Comment (DominikPanic) [inline code/game/objects/items/stacks/medical.dm]: Same line control statement.

File: code/game/objects/items/stacks/medical.dm [hunk 5]
```diff
@@ -147,6 +147,27 @@
 	stack_id = "Tear petals"
 
 
+/obj/item/stack/medical/advanced
+	dir = NORTH
+	flags_atom = DIRLOCK
+
+/obj/item/stack/medical/advanced/update_icon()
+	if(max_amount < 1)
```
Comment (DominikPanic) [inline code/game/objects/items/stacks/medical.dm]:
```suggestion
	if(max_amount < 1 || amount < max_amount)
```

File: code/game/objects/items/stacks/medical.dm [hunk 6]
```diff
@@ -147,6 +147,27 @@
...
+		return
+	var/percentage = round(amount / max_amount) * 100
+	if(amount < max_amount)
```
Comment (DominikPanic) [inline code/game/objects/items/stacks/medical.dm]:
```suggestion

```

**Pull request merged**